code/modules/admin/admin_verbs.dm

```diff
@@ -675,23 +675,19 @@ var/list/admin_verbs_mentor = list(
 	set desc = "Cause an explosion of varying strength at your location."
 
 	var/turf/epicenter = mob.loc
-	var/list/choices = list("Small Bomb", "Medium Bomb", "Big Bomb", "Huge Bomb", "Colossal Bomb", "Apocalyptic Bomb", "Custom Bomb")
```

Comment:
why would you do this, I added these myself, they're for largescale testing

Reply:
please revert these changes, this is an admin tool

Reply:
@Snypehunter007 asked me to remove them.

Reply:
And also, use custom bomb. Jfc.

Reply:
Third note: This was requested by Snype. I adhere to his requests.

Reply:
i'll talk to snype about it, put them back

Reply:
18:17] Snypehunter007: Thats fine then.

As agreed upon. Resolved.